site/content/docs/02-template-syntax.md

```diff
@@ -205,6 +205,8 @@ Iterating over lists of values can be done with an each block.
 </ul>
 ```
 
+You can use each blocks to iterate over any value with a `length` property.
+
```

Comment:
Technically, its any `array-like` object, which also implies that the index-based getter exists as well.

Reply:
I didn't feel like trying to find the words to explicitly explain how elements were accessed :-x